catalog.html

```diff
@@ -2,236 +2,240 @@
-                            <label for="filter-responsive">Резиновая</label>
-                        </li>
-                    </ul>
-                </fieldset>
+        <fieldset>
+          <legend>Особенности:</legend>
+          <ul>
+            <li>
+              <input class="filter-input-checkbox" type="checkbox" name="slider" id="filter-slider" checked>
+              <label for="filter-slider">Слайдер</label>
+            </li>
+            <li>
+              <input class="filter-input-checkbox" type="checkbox" name="advantages" id="filter-advantages">
+              <label for="filter-advantages">Блок преимуществ</label>
+            </li>
+            <li>
+              <input class="filter-input-checkbox" type="checkbox" name="news" id="filter-news" checked>
+              <label for="filter-news">Новости</label>
+            </li>
+            <li>
+              <input class="filter-input-checkbox" type="checkbox" name="gallery" id="filter-gallery">
+              <label for="filter-gallery">Галерея</label>
+            </li>
+            <li>
+              <input class="filter-input-checkbox" type="checkbox" name="cart" id="filter-cart">
+              <label for="filter-cart">Корзина</label>
+            </li>
+          </ul>
+        </fieldset>
+        <button class="filter-button" type="submit">Показать</button>
+      </form>
+    </section>
 
-                <fieldset>
-                    <legend>Особенности:</legend>
-                    <ul>
-                        <li>
-                            <input type="checkbox" name="slider" id="filter-slider" checked>
-                            <label for="filter-slider">Слайдер</label>
-                        </li>
-                        <li>
-                            <input type="checkbox" name="advantages" id="filter-advantages">
-                            <label for="filter-advantages">Блок преимуществ</label>
-                        </li>
-                        <li>
-                            <input type="checkbox" name="news" id="filter-news" checked>
-                            <label for="filter-news">Новости</label>
-                        </li>
-                        <li>
-                            <input type="checkbox" name="gallery" id="filter-gallery">
-                            <label for="filter-gallery">Галерея</label>
-                        </li>
-                        <li>
-                            <input type="checkbox" name="cart" id="filter-cart">
-                            <label for="filter-cart">Корзина</label>
-                        </li>
-                    </ul>
-                </fieldset>
-                <button type="submit">Показать</button>
-            </form>
-        </section>
+    <section class="sorting">
+      <h2 class="sorting-title">Сортировать:</h2>
+      <ul class="sorting-list">
+        <li class="sorting-item-current">
```

Comment:
Обычно класс, который указывает на выбранный элемент, встает вторым классом к основному:

```suggestion
        <li class="sorting-item sorting-item-current">
```

Кстати, в пагинации так и сделано )